cfme/tests/services/test_dialog_element_in_catalog.py

```diff
@@ -59,8 +59,59 @@ def test_dropdownlist_required_dialog_element(appliance, catalog_item):
     Polarion:
         assignee: nansari
         initialEstimate: 1/4h
+        testtype: functional
         casecomponent: Services
+        startsin: 5.10
     """
     service_catalogs = ServiceCatalogs(appliance, catalog_item.catalog, catalog_item.name)
     view = navigate_to(service_catalogs, 'Order')
     assert view.submit_button.disabled
+
+
+@pytest.mark.manual
+@pytest.mark.tier(1)
+@pytest.mark.parametrize('element_type', ['text_box', 'checkbox', 'text_area', 'radiobutton',
+                                          'date_picker', 'timepicker', 'tagcontrol'])
+def test_required_dialog_elements(element_type):
```

Comment:
Isn't this already covered by automation with the `dropdown_dialog` fixture in this module? I realize this covers more types than just dropdown, but why not automate this right away? The framework components should all be available, its much more worth your time.

Reply:
@mshriver Right now, i have dynamic dialog FA(new automation and customer BZs) on higher priority, i will definitely automate it later some time. For tracking purpose let this PR gets merged.